modules/cleanmedianetBidAdapter.js

```diff
@@ -0,0 +1,307 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { config } from '../src/config';
+import { Renderer } from '../src/Renderer';
+import { BANNER, VIDEO } from '../src/mediaTypes';
+
+export const helper = {
+  getTopFrame: function() {
```

Comment:
Please use bidderRequest.refererInfo for getting the url
http://prebid.org/dev-docs/bidder-adaptor.html#bidrequest-parameters

Reply:
Fixed